### PR TITLE
feature: add referer middleware

### PR DESCRIPTION
**Pull request by rtirserio**
adds middleware for checking the referrer. if no allowed referrers are configured, the middleware is not used

### CODE REVIEW

**Comment by mjoyce91** [comment]
Codeclimate failures?

File: src/server.js [hunk 1]
```diff
@@ -19,6 +20,9 @@ const API_ROOT = process.env.API_ROOT || 'http://localhost:8000';
 // define the prefix for the application
 const PUBLIC_URL = process.env.PUBLIC_URL || '/talentmap/';
 
+// allowd referers
```
Comment (burgwyn) [inline src/server.js]: Typo

Comment (rtirserio) [reply src/server.js]: fixed

**Comment by rtirserio** [comment]
@mjoyce91 what failure are you referring to? the CI fails are due to the branch name having a / in it. we didnt move that to the mvp1 branch

**Comment by burgwyn** [comment]
Hold off on merging

**Comment by rtirserio** [comment]
closing. we are going to use this

**Pull request closed without merging**